content/community/resources/bees/contents.lr

```diff
@@ -10,7 +10,7 @@ Cross-platform application development is a holy grail of software engineering.
 ---
 event_name: GitHub Constellation Sydney
 ---
-event_url: githubconstellation.com/sydney/
+event_url: https://githubconstellation.com/sydney/
```

Comment:
Hi @glasnt it should be

http://githubconstellation.com/sydney/  (http, without the s :-p)